ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -245,7 +245,7 @@ - (void)updateCacheToResult:(ADAuthenticationResult *)result
            || ![ADAuthenticationContext handleNilOrEmptyAsResult:item argumentName:@"resource" authenticationResult:&result]
            || ![ADAuthenticationContext handleNilOrEmptyAsResult:item argumentName:@"accessToken" authenticationResult:&result])
         {
-            AD_LOG_WARN(@"Told to update cache to an invalid token cache item", [context correlationId], nil);
+            AD_LOG_WARN(nil, @"Told to update cache to an invalid token cache item.");
```

Comment:
why did you remove correlationId logging?

Reply:
fixed